portal/templates/portal/play.html

```diff
@@ -10,7 +10,7 @@
     <script type="text/javascript" src="{% static 'portal/js/passwordStrength.js' %}"></script>
     <script>
     var PASSWORD_FIELD_ID = '{{ signup_form.password.auto_id }}';
-    var PASSWORD_TYPE = 'STUDENT';
+    var PASSWORD_TYPE = 'INDEPENDENT_STUDENT';
```

Comment:
A student who is registered at a school or club gets a password issued to them by the teacher, and then logs in directly with that password. An independent student has to register, therefore creating their own password, so this password strength functionality is only used for independent students.